src/css/styles/_c-form.scss

```diff
@@ -0,0 +1,177 @@
+.c-form {
+  position: absolute;
```

Comment:
![image](https://user-images.githubusercontent.com/789841/46853667-0d66de00-cdff-11e8-9c87-8d522e06300b.png)

when I open dev tools it jumps to the top

- [ ] always on bottom

Reply:
I think it is an issue with dev tools and interpreting resize of a window in some situations. I was trying to find solution for that, but nothing better came up. In normal circumstances it works properly.